src/ntfs/write.rs

```diff
@@ -179,6 +175,48 @@ pub fn write_fares_v1(
     Ok(())
 }
 
+fn write_fares_v1_from_v2(
+    base_path: &path::Path,
+    tickets: &CollectionWithId<Ticket>,
```

Comment:
I wonder if it would be useful to create a struct `Fares` (or `FaresV2`) that would hold the five different collections afferent to fares. Something like
```rust
struct Fares {
	tickets: &CollectionWithId<Ticket>,
   ticket_uses: &CollectionWithId<TicketUse>,
   ticket_prices: &Collection<TicketPrice>,
   ticket_use_perimeters: &Collection<TicketUsePerimeter>,
   ticket_use_restrictions: &Collection<TicketUseRestriction>,
}
```

It would simplify the signature of our functions.
```rust
fn write_fares_v1_from_v2(
    base_path: &path::Path,
    fares: &Fares,
) -> Result<()>
```